datahub_sap_hana/ingestion.py

```diff
@@ -95,6 +98,9 @@ def create(cls, config_dict: Dict[str, Any], ctx: PipelineContext) -> "HanaSourc
         config = HanaConfig.parse_obj(config_dict)
         return cls(config, ctx)
 
+    def make_dataset_urn(self, dataset_name):
+        return builder.make_dataset_urn(name=dataset_name, platform="hana", env="PROD")
```

Comment:
this will break things, do not hardcode the `env` value, the platform name and the env should come from the config